composer.json

```diff
@@ -9,15 +9,20 @@
         }
     ],
     "require": {
-        "php": "^5.3.9 || ^7.0",
-        "symfony/filesystem": "^2.3 || ^3.0"
+        "php": "^7.1",
```

Comment:
why?

Reply:
requirement for phpunit

Reply:
This lib supports PHP 5 :)

Reply:
Please keep in mind that there is no value to force a PHP requirement bump, especially when there's no need in the lib itself.

If you do want to bump the PHP version, you should do it in a major release.

Reply:
Alright!

Reply:
//push